contracts/storage/VolumeRestrictionTMStorage.sol

```diff
@@ -45,10 +45,15 @@ contract VolumeRestrictionTMStorage {
     VolumeRestriction public defaultRestriction;
     // Daily global restriction that applies to all token holders (Total ST traded daily is restricted)
     VolumeRestriction public defaultDailyRestriction;
-    // Restriction stored corresponds to a particular token holder
-    mapping(address => VolumeRestriction) public individualRestriction;
-    // Daily restriction stored corresponds to a particular token holder
-    mapping(address => VolumeRestriction) public individualDailyRestriction;
+
+    struct Restrictions {
+        // Restriction stored corresponds to a particular token holder
+        mapping(address => VolumeRestriction) individualRestriction;
+        // Daily restriction stored corresponds to a particular token holder
+        mapping(address => VolumeRestriction) individualDailyRestriction;
+    }
+
+    Restrictions restrictions;
```

Comment:
Does this actually save some size?

Reply:
Not really, but does tidy things up a bit.